### PR TITLE
Optimisation to Surface Duo update

### PR DESCRIPTION
**Pull request by thewulf7**
General fixes for #3 + fixes for latest emulator version

- Changed approach to retrive isSpanning and getWindowRects methods
- Simplified code of DualScreenInfo.kt
- Updating naming to RN convention
- Fixed iOS stub
- Fixed examples

### CODE REVIEW

File: dualscreeninfo/package.json [hunk 1]
```diff
@@ -36,20 +36,19 @@
...
-    "@types/react-native": "^0.61.10",
+    "@types/react": "^16.8.6",
+    "@types/react-native": "^0.61.17",
     "babel-jest": "^25.1.0",
     "jest": "^25.1.0",
     "metro-react-native-babel-preset": "^0.58.0",
     "prettier": "^1.19.1",
-    "react": "16.8.6",
-    "react-native": "^0.60.6",
     "react-native-windows": "^0.60.0-vnext.127",
```
Comment (kmelmon) [inline dualscreeninfo/package.json]: >"react-native-windows": "^0.60.0-vnext.127", [](start = 4, length = 44)

react-native-windows version 60 isn't compatible with react-native  version 61.  This should be updated to ^0.61.0-beta.28

File: dualscreeninfo/package.json [hunk 2]
```diff
@@ -36,20 +36,19 @@
...
-    "react-native-windows":  "*"
+    "react-native-windows": "*"
   },
   "devDependencies": {
     "@babel/core": "^7.8.4",
     "@babel/preset-env": "^7.8.4",
     "@react-native-community/bob": "^0.9.7",
     "@types/jest": "^25.1.1",
-    "@types/react-native": "^0.61.10",
+    "@types/react": "^16.8.6",
```
Comment (kmelmon) [inline dualscreeninfo/package.json]: >"@types/react": "^16.8.6", [](start = 4, length = 26)

I think this should be updated to 16.9.0

File: dualscreeninfo/src/dualscreeninfo/index.ts [hunk 3]
```diff
@@ -3,30 +3,48 @@
...
-    addEventListener: (
-        type: DualScreenInfoEvent,
-        handler: SpannedChangeHandler
-    ) => void;
-    removeEventListener: (
-        type: DualScreenInfoEvent,
-        handler: SpannedChangeHandler
-    ) => void;
+	  isSpanning: boolean;
+		windowRects: WindowRect[];
```
Comment (kmelmon) [inline dualscreeninfo/src/dualscreeninfo/index.ts]: nit-pick:  mixing space/tabs.  Note that on our team we prefer spaces to tabs.  I don't feel too strongly about this as long as it's consistent.

File: dualscreeninfo/package.json (hunk 1 above)
Comment (thewulf7) [reply dualscreeninfo/package.json]: fixed

File: dualscreeninfo/package.json (hunk 2 above)
Comment (thewulf7) [reply dualscreeninfo/package.json]: fixed

File: dualscreeninfo/src/dualscreeninfo/index.ts (hunk 3 above)
Comment (thewulf7) [reply dualscreeninfo/src/dualscreeninfo/index.ts]: fixed

**Pull request merged**